prow/apis/prowjobs/register.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package prowjobs
 
+// GroupName : TODO: (alisondy) Investigate usage of GroupName
```

Comment:
I'm not terribly convinced that adding a bunch of Todo comments is particularly useful. 

At least now we know what needs documentation because it adds a lint error. 

This silences the lint error without actually fixing the reason for the lint error. Meaning no one will ever fix it.

Reply:
Yes, that sounds reasonable, I will go through and try resolve these todo's before merging in

Reply:
Sweet thanks! ❤️

Reply:
Feel free to drop a comment on parts that are confusing if you want someone else to chime in. I'm sure we can either answer and/or know the right person who can

Reply:
sure thing ✨